docs/development.md

```diff
@@ -237,24 +246,29 @@ manually `source`'d in a shell.
 Torch-MLIR can also be built using Bazel (apart from the official CMake build) for users that depend on Bazel in their workflows. To build `torch-mlir-opt` using Bazel, follow these steps:
 
 1. Launch an interactive docker container with the required deps installed:
+
 ```shell
 ./utils/bazel/docker/run_docker.sh
 ```
```

Comment:
Thanks for adding the newline before the code block.

You should also indent these code blocks so they appear nested under the list items:

```suggestion
1. Launch an interactive docker container with the required deps installed:

    ```shell
    ./utils/bazel/docker/run_docker.sh
    ```
```

First here is with the indentation, second is without:
![image](https://github.com/user-attachments/assets/563d78f0-a029-4dc5-910e-1269a05c791f)